res_syntax/src/res_uncurried.ml

```diff
@@ -1,20 +1,16 @@
-type config = Legacy | Default | Always
-
-let init = ref Legacy
-
-let isDefault = function
-  | Legacy -> false
-  | Default -> true
-  | Always -> true
+let isSwap = function
```

Comment:
The naming might be a bit confusing here, one would expect `isSwap` to return `true` for the `Swap` case only.